CMSgov/design-system
# [WMNGDS-722] Update Banner scss

### Changed
- Refactor banner scss
- Update svg files U.S Flag and Locked padlock to have aria-labelledby for accessibility
- Update banner examples and test snap-shot 

### How to test
- Run the doc site locally with `yarn start` to test the Banner component (just noticed that we're not able to test on the demo site for this as the image file are not being picked up correctly on the demo site)
- Ensure banner works on various browsers and is accessible by screen readers
